modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+* Non-Repeatable Stream | Configure if repeatable streams should be used and their behavior. |  | 
+| Target Variable | String | The name of a variable on which the operation output is placed. |  | 
+| Target Value | String | An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config]
+
+=== Serialize
+
+`<graphql-router:serialize>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Inbound Content | Any | Content of the message (usually, the payload). | #[payload] | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Object Type Name | String | The object-type name. |  | *x*
+| Field Name | String | The field name. |  | 
+| Target Variable | String | The name of a variable on which the operation output is placed. |  | 
+| Target Value | String | An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config]
+
+=== Serialize Batch
+
+`<graphql-router:serialize-batch>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String | The object-type name. |  | *x*
+| Inbound Content | Any | Content of the message (usually, the payload). | #[payload] | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Target Variable | String | The name of a variable on which the operation output is placed. |  | 
+| Target Value | String | An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config]
+
+== Sources
+
+=== Data Fetcher
+
+`<graphql-router:data-fetcher>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Output MIME Type | String | The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding | String | The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
```

Comment:
The only is unclear to me. Is it only executed? only executed on primary node? only when running in a cluster?

Reply:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22only%22(%22simply,%22%22just%22)

Reply:
If false, the execution of the source will be done anywhere in the cluster. If true, it forces to run only in the primary node.
Just to clarify, these tables are autogenerated from the connector and JDK javadoc, so the same phrases are spread along several reference documentations.

Reply:
Taking into account what Damian said (the phrases on this table are used in other reference documentation), and that the description explains that the "only" belongs to the action of being executed (only executed in the primary node), let's leave this reference as it has been generated.
Thanks!